packages/utils/telemetry-utils/src/eventEmitterWithErrorHandling.ts

```diff
@@ -11,22 +11,8 @@ import { TypedEventEmitter, EventEmitterEventType } from "@fluidframework/common
  * Any exception thrown by "error" listeners will propagate to the caller.
  */
 export class EventEmitterWithErrorHandling<TEvent extends IEvent = IEvent> extends TypedEventEmitter<TEvent> {
-    private readonly errorHandler: (eventName: EventEmitterEventType, error: any) => void;
-
-    constructor(errorHandler?: (eventName: EventEmitterEventType, error: any) => void) {
+    constructor(private readonly  errorHandler: (eventName: EventEmitterEventType, error: any) => void) {
         super();
-        this.errorHandler = errorHandler ?? this.defaultErrorHandler.bind(this);
-    }
-
-    private defaultErrorHandler(event, error) {
-        // Some listener threw an error, we'll try emitting that error via the error event
-        // But not if we're already dealing with the error event, in that case just let the error be thrown
-        if (event === "error") {
-            throw error;
-        }
-
-        // Note: This will throw if no listeners are registered for the error event
-        super.emit("error", error);
```

Comment:
Now that the errorHandler is required during construction, I don't think it's possible for a provided errorHandler to call `emit` (as the instance does not yet exist when the errorHandler is instantiated) as the default handler used to here. Is that an intentional change?

Reply:
I'm not sure I follow. The change in this same PR has no issue accessing this in such handler - you can emit just fine there:

		super((_event, error: unknown) => {
			this.tree.emit('error', error);
		});

Yes, you can't do something like that

		super((_event, error: unknown) => this.someHandler.bind(this));

But that's trivial to solve via capturing _this_ in lambda as above

Reply:
I guess I should have phrased this differently. I was wondering if this is actually an inconsistency in the old default handler. If errors can only be thrown by emit (ostensibly because there are no listeners per the comment), is it inconsistent to call emit again within the handler?